docs/src/migrating.md

```diff
@@ -0,0 +1,27 @@
+Migration
+=========
+
+How to migrate from v0.10 to v0.11
+----------------------------------
+
+v0.11 has a few subtle breaking changes to `eof` and `seekend`.
+
+### `Memory(data::ByteData)`
+
+The `Memory(data::ByteData)` constructor was removed.
+Use `Memory(pointer(data), sizeof(data))` instead.
+
+### `seekend(stream::TranscodingStream)`
+
+Generic `seekend` for `TranscodingStream` was removed.
+Use `skip(stream, typemax(Int64))` instead.
```

Comment:
This seems weird. Why `typemax(Int64)`? Why is that better than the old generic `seekend`?

Reply:
That could be any integer larger than the size of the stream.
After calling `seekend`, `position` should be the total stream size but there is no generic way to get this without processing the whole stream. Ref #183